src/AspireSamples/AspireSamples.Web/Program.cs

```diff
@@ -0,0 +1,47 @@
+using Microsoft.Extensions.AI;
+using AspireSamples.Web.Components;
+using AspireSamples.Web.Services;
+using AspireSamples.Web.Services.Ingestion;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.AddServiceDefaults();
+builder.Services.AddRazorComponents().AddInteractiveServerComponents();
+
+builder.AddOllamaApiClient("chat")
+    .AddChatClient()
+    .UseFunctionInvocation()
+    .UseOpenTelemetry(configure: c =>
+        c.EnableSensitiveData = builder.Environment.IsDevelopment());
+builder.AddOllamaApiClient("embeddings")
+    .AddEmbeddingGenerator();
+
+builder.AddQdrantClient("vectordb");
+builder.Services.AddQdrantVectorStore();
+builder.Services.AddQdrantCollection<Guid, IngestedChunk>(IngestedChunk.CollectionName);
+builder.Services.AddSingleton<DataIngestor>();
+// By default, we ingest PDF files from the /wwwroot/Data directory.
+// Important: ensure that any content you ingest is trusted, as it may be reflected back
+// to users or could be a source of prompt injection risk.
+builder.Services.AddKeyedSingleton("ingestion_directory", new DirectoryInfo(Path.Combine(builder.Environment.WebRootPath, "Data")));
```

Comment:
this is pretty lame way of providing the arguments, I think in the long term it would be best to expand the sample and have one text box with URIs (the reader can ingest them)